packages/mobile/src/navigator/NavigatorWrapper.tsx

```diff
@@ -97,6 +99,15 @@ export const NavigatorWrapper = () => {
     }
   }, [isReady])
 
+  React.useEffect(() => {
+    RNShake.addEventListener('ShakeEvent', () => {
+      navigate(Screens.SupportContact)
+    })
+    return () => {
+      RNShake.removeEventListener('ShakeEvent')
+    }
+  })
```

Comment:
```suggestion
  }, [])
```

Otherwise it's gonna subscribe/unscribe on each render.

Reply:
Whoops, that's really bad from me :(

Reply:
No worry, it's easy to miss, I think there's a lint rule we could add to make sure it's present.